axelor-prestashop/src/main/resources/views/Product.xml

```diff
@@ -0,0 +1,273 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<object-views xmlns="http://axelor.com/xml/ns/object-views"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://axelor.com/xml/ns/object-views http://axelor.com/xml/ns/object-views/object-views_5.0.xsd">
+
+	<form name="product-form" title="Product" model="com.axelor.apps.base.db.Product" id="prestashop-product-form"
```

Comment:
Use panel-include to include view of product-form and add sidebar panel at the end for prestashop fields with title 'Prestashop'